src/main/java/io/stargate/sgv2/jsonapi/service/shredding/model/DocValueHasher.java

```diff
@@ -198,6 +198,8 @@ public DocValueHash getHash(Object value) {
       return numberValue((BigDecimal) value).hash();
     } else if (value instanceof Boolean) {
       return booleanValue((Boolean) value).hash();
+    } else if (value instanceof Date) {
```

Comment:
Can we remove this? Cassandra uses Instance

Reply:
(I assume you mean `Instant`)

I'll see if I can change test to not need it.

Reply:
Changed.